src/MAUI/Maui.Samples/Samples/Layers/AddDynamicEntityLayer/AddDynamicEntityLayer.xaml

```diff
@@ -0,0 +1,59 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage x:Class="ArcGIS.Samples.AddDynamicEntityLayer.AddDynamicEntityLayer"
+             xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Maui;assembly=Esri.ArcGISRuntime.Maui">
+    <Grid Style="{DynamicResource EsriSampleContainer}">
+        <esriUI:MapView x:Name="MyMapView" Style="{DynamicResource EsriSampleGeoView}" />
+        <Border x:Name="MainUI"
+                Style="{DynamicResource EsriSampleControlPanel}"
```

Comment:
On iPhone Simulator (SE 3 and 14 Pro Max), if you start the sample in vertical mode, then rotate to horizontal mode, the control panel takes up the entire screen. If you start sample in horizontal mode, it'll display the control panel and map just fine. However, upon rotating the screen to vertical mode and back to horizontal mode, the map view is invisible.